apps/e2e/integration/supplier_order.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { test, expect } from "@playwright/test";
+
+import { baseURL } from "./constants";
+import { getDbHandle } from "@/helpers";
+import { addBooksToCustomer, createSupplierOrder } from "@/helpers/cr-sqlite";
+import { testOrders } from "@/helpers/fixtures";
+
+test.beforeEach(async ({ page }) => {
+	await page.goto(`${baseURL}orders/suppliers/`);
+});
+
+testOrders("should create a new supplier order", async ({ page, supplier, books }) => {
```

Comment:
"should allow a new supplier order to be placed from a batch of possible customer order lines" (or more detail to that effect)